GameOfLife.java

```diff
@@ -84,31 +130,57 @@ private static int[][] evolve(int[][] board) {
 	// Assumes that i is at least 1 and at most the number of rows in the board - 1. 
 	// Assumes that j is at least 1 and at most the number of columns in the board - 1. 
 	// Uses the count(board,i,j) function to count the number of alive neighbors.
-	private static int cellValue(int[][] board, int i, int j) {
-		//// Replace the following statement with your code.
+	public static int cellValue(int[][] board, int i, int j) 
+	{
+		int sumelivecell = count(board,i,j);
```

Comment:
use camel case to name variables.
```suggestion
		int sumLiveCell = count(board,i,j);
```